### PR TITLE
S3

### PR DESCRIPTION
**Pull request by jgockley62**
S3 Bucket Request

### CODE REVIEW

File: config/prod/JKG-s3.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  #LifecycleDataTransition: '90'
+  # (Optional) Number of days (from creation) when objects are deleted from S3 and LifecycleDataStorageClass, default is 365000
+  #LifecycleDataExpiration: '1825'
+hooks:
+  before_create:
+    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/s3-bucket.yaml --create-dirs -o templates/remote/s3-bucket.yaml"
+  before_update:
+    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/aws-infra/master/s3-bucket.yaml --create-dirs -o templates/remote/s3-bucket.yaml"
+  after_create:
+    - !synapse_bucket_notify {{stack_group_config.aws_account_name}} {{stack_group_config.aws_account_email}}
```
Comment (zaro0508) [inline config/prod/JKG-s3.yaml]: sorry, the example PR was slightly out of date.  we have updated.  please replace `!synapse_bucket_notify` with `!s3_notify`

File: config/prod/JKG-s3.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template_path: remote/s3-bucket.yaml
+stack_name: JKG-Bucket
```
Comment (zaro0508) [inline config/prod/JKG-s3.yaml]: We prefer this name to match the filename.  could you please either change this to `JKG-s3`? or change the filename to `JKG-Bucket.yaml`?

Comment (jgockley62) [reply config/prod/JKG-s3.yaml]: ```remote/JKG-Bucket.yaml```
or just
```JKG-Bucket.yaml```

Comment (zaro0508) [reply config/prod/JKG-s3.yaml]: remote/JKG-Bucket.yaml

**Pull request closed without merging**